src/org/omegat/Main.java

```diff
@@ -251,10 +252,36 @@ private static void applyConfigFile(String path) {
         }
     }
 
+    protected static void setDefaultTheme() {
+        try {
+            UIManager.setLookAndFeel(UIManager.getSystemLookAndFeelClassName());
+        } catch (Exception ex) {
+            // Something went wrong!!
+            Log.log(ex);
+            showError(ex);
+        }
+    }
+
+    protected static String setTheme(String theme) {
+        try {
+            UIManager.setLookAndFeel(UIDesignManager.getThemeClassName(theme));
+            return theme;
+        } catch (Exception e) {
+            Log.log(e);
+            setDefaultTheme();
+            return Preferences.THEME_DEFAULT;
+        }
+    }
+
```

Comment:
This seems like maybe it should move into `UIDesignManager`.

Reply:
done.